src/components/views/voip/CallView.tsx

```diff
@@ -482,7 +485,7 @@ export default class CallView extends React.Component<IProps, IState> {
 
         // This is a bit messy. I can't see a reason to have two onHold/transfer screens
         if (isOnHold || transfereeCall) {
-            if (isVideoCall) {
+            if (call.hasLocalUserMediaVideoTrack || call.hasRemoteUserMediaVideoTrack) {
```

Comment:
this looks like the perfect place to use `call.type === CallType.Video` for readability